.github/workflows/docs.yml

```diff
@@ -93,6 +93,8 @@ jobs:
 
       - name: Build Documentation
         run: make -C doc html SPHINXOPTS="-W --keep-going"
+        env:
+          PYTHONWARNINGS: EGLWarning
```

Comment:
1. I think this should be something like `'ignore::EGLWarning'`
2. I would prefer to set this envvar in `doc/conf.py` so that local doc builds benefit from the same ignore, and show the same behaviour as CI.

Reply:
I am +1 for "2.". #2745 is more explicit than writing to GitHub Actions.

Reply:
Done in 7d3c664187e5d25de2a6e11947ac4efa6203e2fa.